druid-derive/tests/ui.rs

```diff
@@ -0,0 +1,14 @@
+use trybuild::TestCases;
```

Comment:
I would include a link to `trybuild` in the source here, and maybe a really quick summary of how it works. Basically: if some future update to rust brings slightly different error messages (breaking these tests) it would be nice if the person who came to look at this file to figure out the breakage had as easy a time as possible figuring out what they need to do.

Reply:
Sure. Will look into it when I get back.